content/en/infrastructure/containers/orchestrator_explorer.md

```diff
@@ -140,6 +140,191 @@ All of these columns support sorting, which helps you to pinpoint individual wor
+| **Annotations** are also extracted from [a resource's metadata](https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/). They are generally used to support tooling that aid in cluster management. | `annotation#checksum/configmap:a1bc23d4` |
+| **Fields** are attributes found within the resource manifest. Not all attributes are supported, see [Filterable Fields](#filterable-fields) for more details. | `field#metadata.creationTimestamp:>1699535065` |
+| **Metrics** are added to workload resources (pods, deployments, etc.). You can find resources based on their utilization. To see what metrics are supported, see [Resource Utilization Filters](#resource-utilization-filters). | `metric#cpu_usage_pct_limits_avg15:>80%` |
+| **String matching** is supported by some specific resource attributes, see below.<br>*Note: string matching does not use the key-value format, and you cannoy specify the attribute to match on.* | `"10.132.6.23"` (IP),<br>`"9cb4b43f-8dc1-4a0e"` (UID) or<br>`"web-api-3"` (Name) |
+
+>  ***Notes**:*
+>
+>  - *You might find the same key-value pairs as both a tag and label (or annotation) - this is dependent on how your cluster is configured.*
+>  - In addition to the normal quality comparison (`:`), `field#` and `metric#` filters support the following: `tag:>value`, `:<`, `:>=`, and `:<=`.
+
+The following resource attributes are supported in string matching:
+- `metadata.name`
+- `metadata.uid`
+- IP Addresses found in:
+  - Pods
+  - Nodes (Internal and External)
+  - Services (Cluster, External, and Load Balancer IPs)
+
+This means that you do not need to specify a key to search for a resource by name, or IP.
+
+#### Operators
+
+To combine multiple terms into a complex query, you can use any of the following case sensitive Boolean operators:
+
+| Operator | Description | Example |
+|---|---|---|
+| `AND` | **Intersection**: both terms are in the selected events (if nothing is added, AND is taken by default) | `a AND b`   |
+| `OR` | **Union**: either term is contained in the selected events                                             | `a OR b`   |
+| `NOT` / `-` | **Exclusion**: the following term is NOT in the event (apply to each individual raw text search) | `a AND NOT b` or<br>`a AND -b` |
+|  `( )` | **Grouping:** specify how to group terms logically. | `a AND (b OR c)` or<br>`(a AND b) or c` |
+
+### Wildcards
+
+You can use `*` wildcards as part of a term value to filter by partial matches. Some examples:
+
+- `kube_job:stats-*`: Find all resources with a `kube_deployment` tag value starting with `stats-`
+- `pod_name:*canary`: Find all resources with a `pod_name` value ending in `canary`
+- `kube_deployment:*writer*cell-2`: Find resources with `writer` in the middle of the value, ending with `cell-2`
```

Comment:
this one example feels a bit specific 😬 could we change to something more generic? like `kube_deployment:*stat*canary`